Build Error

Latest commit has an issue with the `npm build` step.

Repro: `docker build -t docinator .`

Expected: docker container!

Actual:
```
#20 10.14 src/cli.ts(26,3): error TS2339: Property 'option' does not exist on type 'void'.
#20 10.14 src/cli.ts(39,3): error TS2339: Property 'option' does not exist on type 'void'.
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -12,19 +12,18 @@ COPY docs/ docs
 
 RUN set -e && \
     export NPM_CONFIG_PREFIX=~/.npm-global && \
-    npm install --unsafe-perm 
+    npm install --unsafe-perm && \
+	npm run build
 
 # 
 # Build the host container
 
 FROM openjdk:15-jdk-alpine
 
-COPY --from=builder /docinator /docinator
-
-WORKDIR /docinator
-
 RUN apk add --no-cache nodejs npm graphviz ttf-droid bash ttf-droid-nonlatin
-RUN npm run build
+
+COPY --from=builder /docinator /docinator
+WORKDIR /docinator
 RUN npm install -g .
 
 WORKDIR /data

--- a/package.json
+++ b/package.json
@@ -3,7 +3,7 @@
 	"description": "Extracts documentation from your project to produce a beautiful static website",
 	"license": "Apache-2.0",
 	"name": "@tmobile/docinator",
-	"version": "0.4.4",
+	"version": "0.4.5",
 	"scripts": {
 		"build": "tsc && cp -R src/hugo/site-template dist/hugo/site-template && npm link --force && npm prune && npm pack",
 		"build:clean": "npm run clean && npm run build",

--- a/src/cli-commander-async.ts
+++ b/src/cli-commander-async.ts
@@ -1,4 +1,4 @@
-import { CommanderStatic } from "commander";
+import commander, { CommanderStatic } from "commander";
 
 const promises: Promise<unknown>[] = [];
 
@@ -10,7 +10,7 @@ type CommandInfo = {
 	name?: string;
 };
 
-export type RegisterActions = { to: (commander: CommanderStatic) => void };
+export type RegisterActions = { to: (commander: CommanderStatic) => commander.Command };
 
 /**
  * Registers an async action with commander so that the action can be awaited

